docs/modules/getting_started/pages/flow-deep-dive.adoc

```diff
@@ -0,0 +1,42 @@
+:imagesdir: ../assets/images
+[[deep-dive-dash]]
+= Flow Deep-Dive dashboard
+
+The {page-component-title} plugin includes a pre-designed flow deep-dive dashboard that displays flow data.  
+The dashboard can help with quality of service (QoS) monitoring and includes panels that display data in the following categories:
+
+* flow interface statistics (throughput and data use by application)
+* SNMP interface statistics (interface throughput and errors/discards)
+* conversations (throughput and data use by conversation)
+* hosts (throughput and data use by host)
+* DCSPs (throughput and data use by Differentiated Services Code Points)
+
+== QoS options
+
+The dashboard exposes nodes and interfaces with flows, that you can filter on in the *Node* and *Interface* drop-down lists.  
+
+You can also filter flows traffic by ToS/DSCP and ECN (explicit congestion notification) for nodes and interfaces with these values. 
+
+Select a DSCP value (or combination of values) from the list for the QoS you want to visualize (class selector, IP precedence, assured forwarding, expedited forwarding). 
+
+The ECN drop-down lets you further filter on this value to see which flows use ECN-capable transports, and congestion reports for them.  
+ECN values appear in the *Data Usage by Application/Conversation/Host/DSCP* panels.
+This works for aggregated and unaggregated flow data. 
+
+[[deep-dive-add]]
+== Add a Flow Deep-Dive dashboard
```

Comment:
There is only one pre-designed flow deep-dive dashboard. Therefore I would say:

== Add the pre-designed flow deep-dive dashboard

Reply:
You don't have to "add" the deep dive dashboard.  It's loaded by default, once the plugin is enabled.

Reply:
@mmahacek Thanks. I thought on a fresh install one still had to "add" it. There is a Re-import option. Perhaps I should add a warning if people want to re-import it that they would lose any customization they made to the original one?